2018-11-08 10:06:39[11] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:06:39
Title=Field 'CommentStatus' doesn't have a default value
Message=Message:Field 'CommentStatus' doesn't have a default value
RequestParm:{"OrderCode":null,"DoctorCode":"D201811060000001","ServiceCode":"S201811060000004","CustomerCode":"C201811070000001","Name":"孙","Mobile":"123","Gender":1,"Age":12,"LocatedCity":"上海","Description":"test","ExpectTime":"2018-11-08T00:00:00","CouponCode":null,"OriginAmount":0.0,"OrderAmount":0.0,"DiscountAmount":0.0,"AmountType":2,"DepositAmount":0.0,"DiscountFlg":false,"ImgList":null,"UserID":34,"LevelID":0,"WechatOpenID":null,"PayAmount":0.0}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteNonQueryInternal()
   在 BLToolkit.Data.DbManager.ExecuteNonQuery()
   在 DAL.OpeServiceOrder_DAL.AddServiceOrder(AddServiceOrder_Model model) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 482
   在 BLL.OpeServiceOrder_BLL.AddServiceOrder(AddServiceOrder_Model model) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 58
   在 WebApi.Controllers.Touch.OrderController.AddServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 217
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/AddServiceOrder
Header=Content-Length:450
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:CFFEBC847EA3A84F8D437E418B459606
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:ADDSERVICEORDER
TI:2018/11/8 10:06:39


2018-11-08 10:09:37[7] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:09:37
Title=Field 'CommentStatus' doesn't have a default value
Message=Message:Field 'CommentStatus' doesn't have a default value
RequestParm:{"OrderCode":null,"DoctorCode":"D201811060000001","ServiceCode":"S201811060000004","CustomerCode":"C201811070000001","Name":"孙","Mobile":"123","Gender":1,"Age":12,"LocatedCity":"上海","Description":"test","ExpectTime":"2018-11-08T00:00:00","CouponCode":null,"OriginAmount":0.0,"OrderAmount":0.0,"DiscountAmount":0.0,"AmountType":2,"DepositAmount":0.0,"DiscountFlg":false,"ImgList":null,"UserID":34,"LevelID":0,"WechatOpenID":null,"PayAmount":0.0}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteNonQueryInternal()
   在 BLToolkit.Data.DbManager.ExecuteNonQuery()
   在 DAL.OpeServiceOrder_DAL.AddServiceOrder(AddServiceOrder_Model model) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 482
   在 BLL.OpeServiceOrder_BLL.AddServiceOrder(AddServiceOrder_Model model) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 58
   在 WebApi.Controllers.Touch.OrderController.AddServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 217
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/AddServiceOrder
Header=Connection:Keep-Alive
Content-Length:450
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:CFFEBC847EA3A84F8D437E418B459606
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:ADDSERVICEORDER
TI:2018/11/8 10:08:53


2018-11-08 10:13:29[24] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:13:29
Title=You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
Message=Message:You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
RequestParm:{"CustomerCode":"C201811070000001","flag":2,"tag":0}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReader(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal()
   在 BLToolkit.Data.DbManager.ExecuteListInternal[T](IList`1 list, Object[] parameters)
   在 BLToolkit.Data.DbManager.ExecuteList[T]()
   在 DAL.OpeServiceOrder_DAL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 155
   在 BLL.OpeServiceOrder_BLL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 37
   在 WebApi.Controllers.Touch.OrderController.GetServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 67
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/GetServiceOrder
Header=Content-Length:52
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:5DECAA8B8700F46C1B6EE02E0588EB01
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:GETSERVICEORDER
TI:2018/11/8 10:13:29


2018-11-08 10:13:32[34] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:13:32
Title=You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'A.`PaymentStatus` = 3 AND A.`OrderStatus` = 1 AND A.`CommentStatus` = 1  ORDER B' at line 41
Message=Message:You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'A.`PaymentStatus` = 3 AND A.`OrderStatus` = 1 AND A.`CommentStatus` = 1  ORDER B' at line 41
RequestParm:{"CustomerCode":"C201811070000001","flag":2,"tag":1}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReader(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal()
   在 BLToolkit.Data.DbManager.ExecuteListInternal[T](IList`1 list, Object[] parameters)
   在 BLToolkit.Data.DbManager.ExecuteList[T]()
   在 DAL.OpeServiceOrder_DAL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 155
   在 BLL.OpeServiceOrder_BLL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 37
   在 WebApi.Controllers.Touch.OrderController.GetServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 67
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/GetServiceOrder
Header=Content-Length:52
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:F761B15EDADCCC5DCF2A28C52A84851E
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:GETSERVICEORDER
TI:2018/11/8 10:13:32


2018-11-08 10:13:33[37] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:13:33
Title=You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
Message=Message:You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
RequestParm:{"CustomerCode":"C201811070000001","flag":2,"tag":0}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReader(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal()
   在 BLToolkit.Data.DbManager.ExecuteListInternal[T](IList`1 list, Object[] parameters)
   在 BLToolkit.Data.DbManager.ExecuteList[T]()
   在 DAL.OpeServiceOrder_DAL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 155
   在 BLL.OpeServiceOrder_BLL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 37
   在 WebApi.Controllers.Touch.OrderController.GetServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 67
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/GetServiceOrder
Header=Content-Length:52
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:5DECAA8B8700F46C1B6EE02E0588EB01
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:GETSERVICEORDER
TI:2018/11/8 10:13:33


2018-11-08 10:17:05[6] ERROR index (null) - 
AppName=WebApi
LogTime=2018/11/8 10:17:05
Title=You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
Message=Message:You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')  ORDER BY  A.`CreatetTime` DESC' at line 41
RequestParm:{"CustomerCode":"C201811070000001","flag":2,"tag":0}
TargetSite:Void OnOperationException(BLToolkit.Data.OperationType, BLToolkit.Data.DataException)
Data:
StackTrace:   在 BLToolkit.Data.DbManager.OnOperationException(OperationType op, DataException ex)
   在 BLToolkit.Data.DbManager.HandleOperationException(OperationType op, Exception ex)
   在 BLToolkit.Data.DbManager.ExecuteOperation[T](OperationType operationType, Func`1 operation)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReader(CommandBehavior commandBehavior)
   在 BLToolkit.Data.DbManager.ExecuteReaderInternal()
   在 BLToolkit.Data.DbManager.ExecuteListInternal[T](IList`1 list, Object[] parameters)
   在 BLToolkit.Data.DbManager.ExecuteList[T]()
   在 DAL.OpeServiceOrder_DAL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\DAL\OpeServiceOrder_DAL.cs:行号 155
   在 BLL.OpeServiceOrder_BLL.GetOrderList(String CustomerCode, Int32 tag) 位置 D:\work\BLL\OpeServiceOrder_BLL.cs:行号 37
   在 WebApi.Controllers.Touch.OrderController.GetServiceOrder(JObject obj) 位置 D:\work\WebApi\Controllers\Touch\OrderController.cs:行号 67
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

ProcessName=dyapi
ClientIp=10.0.0.122
ServerIp=10.0.0.122
UrlReferrer=
RawUrl=/Order/GetServiceOrder
Header=Connection:Keep-Alive
Content-Length:52
Content-Type:application/json
Accept-Charset:UTF-8
Authorization:5DECAA8B8700F46C1B6EE02E0588EB01
Expect:100-continue
Host:www.dyapi.com
Accept-Languag:zh-cn
US:0
ME:GETSERVICEORDER
TI:2018/11/8 10:15:33


